docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -320,6 +320,97 @@ will be lost.
 $ kubectl delete -f alluxio-journal-volume.yaml
 ```
 
+#### Upgrade Alluxio
+
+This section will go over how to upgrade Alluxio in your Kubernetes cluster with `kubectl`.
+
+**Step 1: Upgrade the docker image version tag**
+ 
+Each released Alluxio version will have the corresponding docker image released on [dockerhub](https://hub.docker.com/r/alluxio/alluxio).
+
+You should update the `image` field of all the Alluxio containers to use the target version tag. Tag `latest` will point to the latest stable version.
+It is recommended that Alluxio masters and workers are running on the same version for the best compatibility.
+
+For example, if you want to upgrade Alluxio to the latest stable version, update the containers as below:
+
+```yaml
+containers:
+- name: alluxio-master
+  image: alluxio/alluxio:latest
+  imagePullPolicy: IfNotPresent
+  ...
+- name: alluxio-job-master
+  image: alluxio/alluxio:latest
+  imagePullPolicy: IfNotPresent
+  ...
+```
+
+**Step 2: Stop running Alluxio master and worker Pods**
+
+Kill all running Alluxio worker Pods.
+
+```console
+$ kubectl delete -f alluxio-worker.yaml
+```
+
+Then kill all running Alluxio master Pods by killing each StatefulSet and each Service.
+
+```console
+$ let masterCount = {number of your masters minus 1}
+$ for i in {0..masterCount}; do
+> kubectl delete service alluxio-master-{i}
+> kubectl delete statefulset alluxio-master-{i}
+> done
+```
+
+The reason why you don't want to do `kubectl delete -f alluxio-master.yaml` is that will delete the Persistent Volume Claim in the `alluxio-master.yaml` if any.
+And that will result in your Persistent Volume getting released and the journal in it will be lost.
+
+Make sure all the Pods have been terminated before you move on to the next step.
+
+**Step 3: Format journal and Alluxio storage if necessary**
+
+Check the Alluxio upgrade guide page for whether the Alluxio master journal has to be formatted.
```

Comment:
I'm taking the 3rd approach here. I plan to replace "Alluxio upgrade guide page" with the real page once it's ready. The only reason why I'm not taking the 1st option and provide the full picture is because I certainly is not the best person to write that. It may take a considerable time till I finish, and we eventually end up having a good deal of general Alluxio upgrade guide here in this PR.